content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,173 @@
+rates for each account.
+
+To get a developer key and meter number, begin by navigating to FedEx's `Developer Resource Center
+<https://www.fedex.com/en-us/developer/web-services.html>`_. Then, click on the :guilabel:`FedEx Web
+Services` drop-down menu.
+
+Click on :guilabel:`Get Test Key` to start the process of getting a developer key and meter number
+which can be used to configure a shipping method for testing purposes.
+
+Click on :guilabel:`Get Production Key` to start the process of getting a developer key and meter
+number which can be used to configure a shipping method that generates real shipments with FedEx.
+
+After clicking either option, follow the instructions until the :guilabel:`Confirmation` screen is
+reached. This screen displays the developer key and meter number.
+
+Once the developer key and meter number are determined, enter them in the :guilabel:`Developer Key`
+and :guilabel:`Meter Number`fields on the :guilabel:`Fedex Configuration` tab of the shipping
+method.
+
+Shipping details
+~~~~~~~~~~~~~~~~
+
+The main section of the :guilabel:`Fedex Configuration` tab includes a number of additional fields
+used provide information about the shipping method:
+
+- :guilabel:`Fedex Service Type`: the FedEx service used to ship a package.
+- :guilabel:`Fedex Drop-Off Type`: the method for getting a package into FedEx's possession.
+- :guilabel:`Fedex Package Type`: the type of package used for the shipping method.
+- :guilabel:`Package Weight Unit`: the unit of measure used to weigh packages.
+- :guilabel:`Package Length Unit`: the unit of measure used to determine the dimensions of packages.
+- :guilabel:`Label Type`: the type of shipping label used for packages.
+- :guilabel:`Label Format`: the file format used by Odoo to generate shipping labels.
+- :guilabel:`Commercial Invoice Type`: the dimensions and type of the paper used to print invoices.
+
+.. important::
+   The options that should be selected on the :guilabel:`Configuration` tab of a shipping method
+   depend on the negotiated shipping services of the associated FedEx account. To confirm the
+   available services for a FedEx account, visit the :guilabel:`Account Management` page after
+   logging in to the FedEx website, or speak with a customer service representative.
+
+Options
+~~~~~~~
+
+The :guilabel:`Options` section of the :guilabel:`Fedex Configuration` tab provides a few additional
+options to further configure the shipping method:
+
+- :guilabel:`Saturday Delivery`: enable the checkbox to allow packages shipped with the delivery
+  method to be delivered on Saturdays.
+- :guilabel:`Generate Return Label`: enable the checkbox to automatically generate a return label
+  upon validation of a delivery order.
+- :guilabel:`Duties paid by`: use the drop-down menu to select whether duty charges should be paid
+  by the :guilabel:`Sender` or :guilabel:`Recipient`.
+
+Activate shipping method
```

Comment:
Before you get into this, do you think it would be beneficial to (even briefly) discuss the use/importance of the other tabs present on the shipping method form?

Reply:
The other tabs are general to all shipping methods, so they're best covered in the shipping method essentials doc, which I link to in this doc.